Snowflake.Data.Tests/SFPutGetTest.cs

```diff
@@ -107,14 +112,250 @@ public void TearDown()
+                $"{absolutePath}_one.csv",
+                $"{absolutePath}_two.csv",
+                $"{absolutePath}_three.csv"
+            };
+            PrepareFileData(files);
+            // Create file with the same name structure but with a different extension
+            PrepareFileData($"{absolutePath}_four.txt");
+
+            // Set the PUT query variables
+            _inputFilePath = $"{absolutePath}*.csv";
+            _internalStagePath = $"@{_schemaName}.{_stageName}";
+
+            using (var conn = new SnowflakeDbConnection(ConnectionString))
+            {
+                conn.Open();
+                PutFile(conn);
+                // Verify that all files have been uploaded
+                VerifyFilesAreUploaded(conn, files, _internalStagePath);
+            }
+        }
+        
+        [Test]
+        public void TestPutFileQuestionMarkWildcard()
+        {
+            // Prepare the data files to be copied
+            var absolutePath = $"{Path.GetTempPath()}{Guid.NewGuid()}";
+            var files = new List<string> {
+                $"{absolutePath}_1.csv",
+                $"{absolutePath}_2.csv",
+                $"{absolutePath}_3.csv"
+            };
+            PrepareFileData(files);
+            
+            // Set the PUT query variables
+            _inputFilePath = $"{absolutePath}_?.csv";
+            _internalStagePath = $"@{_schemaName}.{_stageName}";
+
+            using (var conn = new SnowflakeDbConnection(ConnectionString))
+            {
+                conn.Open();
+                PutFile(conn);
+                // Verify that all files have been uploaded
+                VerifyFilesAreUploaded(conn, files, _internalStagePath);
+            }
+        }
+        
+        [Test]
+        public void TestPutFileRelativePathWithoutDirectory()
+        {
+            // Set the PUT query variables
+            _inputFilePath = $"{Guid.NewGuid()}_1.csv";
+            _internalStagePath = $"@{_schemaName}.{_stageName}";
+            
+            // Prepare the data files to be copied
+            PrepareFileData(_inputFilePath);
+
+            using (var conn = new SnowflakeDbConnection(ConnectionString))
+            {
+                conn.Open();
+                PutFile(conn);
+                // Verify that all files have been uploaded
+                VerifyFilesAreUploaded(conn, new List<string> { _inputFilePath }, _internalStagePath);
+            }
+        }
+        
+        [Test]
+        public void TestPutFileRelativePathWithDirectory()
+        {
+            var guid = Guid.NewGuid();
+            var relativePath = $"{guid}";
+            Directory.CreateDirectory(relativePath);
+            
+            // Set the PUT query variables
+            _inputFilePath = $"{relativePath}{Path.DirectorySeparatorChar}{guid}_1.csv";
+            _internalStagePath = $"@{_schemaName}.{_stageName}";
+            
+            // Prepare the data file to be copied
```

Comment:
do we need such comments? let's rename a method instead of having a comment like that

Reply:
Removed the comments.